frontend/js/src/explore/fresh-releases/components/ReleaseTimeline.tsx

```diff
@@ -41,6 +43,10 @@ export default function ReleaseTimeline(props: ReleaseTimelineProps) {
         (date) => releasesPerDate[date] >= minReleasesThreshold
       );
 
+      if (sortDirection === "descend") {
+        filteredDates.reverse();
+      }
+
```

Comment:
Here instead of repeating the reverse 4 times in the same functiuon, it would be better to do it a single time right before `return zipObject(percentArr, dataArr);`